packages/jokul/src/components/progress-bar/Countdown.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from "@testing-library/react";
+import React, { act } from "react";
+import { describe, expect, it, vi } from "vitest";
+import { Countdown } from "./Countdown.js";
+
+describe("Countdown", () => {
+    it("renders the countdown with correct initial time", () => {
+        render(<Countdown from={5000} />);
+
+        expect(screen.getByText("5")).toBeInTheDocument();
+    });
+
+    it("counts down correctly", () => {
+        vi.useFakeTimers();
+        render(<Countdown from={5000} />);
+
+        act(() => {
+            vi.advanceTimersByTime(1000);
+        });
+
+        expect(screen.getByText("4")).toBeInTheDocument();
+
+        act(() => {
+            vi.advanceTimersByTime(1000);
+        });
+
+        expect(screen.getByText("3")).toBeInTheDocument();
+    });
+
+    it("pauses the countdown when isPaused is true", () => {
+        vi.useFakeTimers();
+        render(<Countdown from={5000} isPaused={true} />);
+
+        vi.advanceTimersByTime(1000);
+        expect(screen.getByText("5")).toBeInTheDocument();
+    });
+
+    it("stops countdown at zero", () => {
+        vi.useFakeTimers();
+        render(<Countdown from={2000} />);
+
+        act(() => {
+            vi.advanceTimersByTime(2000);
```

Comment:
```suggestion
            vi.advanceTimersByTime(3000);
```

Hvis jeg skal flisespikke litt her så sjekker ikke denne testen av vi stopper på 0, den sjekker at vi **er** på null etter 2 sekunder. For å sjekke at ikke fortsetter til -1 burde vi flytte klokka lenger frem.